journal/original/12-chapter-9.adoc

```diff
@@ -2329,6 +2364,9 @@ that I was moved to give forth the following, as an awakening warning to them:--
 [.small-break]
 '''
 
+[.embedded-content-document.letter]
```

Comment:
This is pretty subjective, but let's change this to an "address" instead of a "letter". And while you're at it, can you remove the `[.small-break]` above as well?